### DISCUSSION TITLE
Help me compile, run and debug dbeaver

### DISCUSSION THREAD

**Discussion by adrmrv** [dbeaver/dbeaver#20110] [DBeaver Development]
Hi,

I followed the wiki, it says CTRL+B to compile dbeaver but it does nothing on my computer
Please explain how to compile, run and debug dbeaver

![image](https://github.com/dbeaver/dbeaver/assets/20116994/a28b8701-f595-41b6-8429-4f8e178bddb5)

**Comment by E1izabeth** [comment]
We can't help you without any information about what you've done, what error do you see and what exactly are you asking about.
